src/pages/LandingPage.js

```diff
@@ -67,20 +62,29 @@ const makeCard = (link, title, body) => h(Clickable, {
   makeRightArrowWithBackgroundIcon()
 ])
 
-const LandingPage = () => {
-  return h(HeroWrapper, { bigSubhead: true }, [
+const getSiteSpecificHyperlinks = () => Utils.cond(
+  [isRareX(), () => [
+    makeDocLink('https://rare-x.org/DataAnalysisPlatform-Documentation',
+      'Find RARE-X Data Analysis Platform documentation, tutorials and Jupyter notebook examples'),
+    makeDocLink('https://rare-x.org/researchers/', 'Learn more about the RARE-X Data Analysis Platform'),
+    makeDocLink('https://support.terra.bio/hc/en-us', 'Find Terra how-to\'s, documentation, video tutorials, and discussion forums')
+  ]],
+  () => [
```

Comment:
Should the links conditional on `isTerra` and `isFirecloud` be separate cases for this `Utils.cond` instead of grouped in the default option?

Reply:
I was on the fence about it. Changed in [5084c9d](https://github.com/DataBiosphere/terra-ui/pull/3212/commits/5084c9df38630c09bbedaf932ac0fbedd764cc04).